packages/scenario/CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # @getodk/scenario
 
+## 0.6.0
+
+### Minor Changes
+
+- 81a57c3: Support for rank question types (`<odk:rank>`)
+- 99295eb: - Support for geopoint questions with no appearance
+  - Support for geopoint notes
+
+### Patch Changes
+
+- 0287a16: Fix: include namespace declarations in submission XML
+- a08e77b: Fix: include primary instance root attributes in submission XML
+- e6d01b0: - Partial support for `<range>` (basic horizontal and vertical sliders)
```

Comment:
```suggestion
- e6d01b0: Partial support for `<range>` (basic horizontal and vertical sliders)
```

Reply:
You're so right, thank you! Added a commit to fix all of them. We'll need to remember not to have a dash for the first line of a changeset message.